streamer/src/recvmmsg.rs

```diff
@@ -22,7 +22,7 @@ pub fn recv_mmsg(socket: &UdpSocket, packets: &mut [Packet]) -> io::Result</*num
     let count = cmp::min(NUM_RCVMMSGS, packets.len());
     for p in packets.iter_mut().take(count) {
         p.meta.size = 0;
-        match socket.recv_from(&mut p.data) {
+        match socket.recv_from(&mut p.buffer_mut()) {
```

Comment:
I think clippy will only complain about this on macos targets, can you drop the `&mut` here?

Reply:
done